src/app/core/api.service.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('xyz-cv-ui.core')
+        .factory('api', api);
+
+    function api($resource, config) {
```

Comment:
(I don't think the consumer of the api service should need to know the path to the resources it needs, also, how does this work with DI?)